packages/perennial/contracts/multiinvoker/MultiinvokerRollup.sol

```diff
@@ -0,0 +1,290 @@
+pragma solidity ^0.8.0;
+
+import "./MultiInvoker.sol";
+
+// import "forge-std/Test.sol";
+
+contract MultiInvokerRollup is MultiInvoker {
+
+    event AddressAddedToCache(address indexed addr, uint256 nonce);
+
+    uint256 public addressNonce;
+    mapping(uint => address) public addressCache;
+    mapping(address => uint) public addressNonces;
+
+    
+    constructor(
+        Token6 usdc_,
+        IBatcher batcher_,
+        IEmptySetReserve reserve_,
+        IController controller_
+    ) MultiInvoker(
+        usdc_,
+        batcher_,
+        reserve_,
+        controller_
+    ) { }
+
+    /// @dev fallback eliminates the need to include function sig
+    fallback (bytes calldata input) external returns (bytes memory) {
+        decodeFallbackAndInvoke(input);
```

Comment:
doesn't return anything

Reply:
fallback must have the signature `fallback()` or `fallback(bytes memory input) returns (bytes memory)`. can return empty bytes if you think thats cleaner